app-misc/mdate/mdate-1.7.0.3.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A converter to Mayan dates"
+DESCRIPTION="Mdate is a utility that converts Gregorian dates to Mayan Long Count dates. \
+It was originally an implementation of some algorithms published by N. Dershowitz and \
+E.M. Reingold in their book Calendrical Calculations."
+HOMEPAGE="https://github.com/ewe2/mdate"
+COPYRIGHT="2010 - 2014 Sean Dwyer"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/ewe2/mdate/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="6d58a8cf63c25b2b4b5b8904ba7c982f18608e8fa65436c555280699a020bcf8"
+PATCHES="mdate-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	mdate = $portVersion
+	cmd:mdate = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:make
+	cmd:gcc
+	cmd:ld
+	"
+
+BUILD()
+{
+	make
+}
+
+INSTALL()
+{
+	make installbin BINDIR=$binDir
+	install -d $docDir $manDir/man1
+	install -d $docDir AUTHORS doc/mdate.html doc/mdate.pdf doc/mdate.txt
+	install -d $manDir/man1 doc/mdate.1
```

Comment:
The first install, in line 40 is ok, but the 2 others, in lines 41 and 42 are incorrect. The correct option for them is "-t", not "-d". Could you fix these 2 tiny errors? Here's a diff-style comparison:
```diff
 	install -d $docDir $manDir/man1
-	install -d $docDir AUTHORS doc/mdate.html doc/mdate.pdf doc/mdate.txt
+	install -t $docDir AUTHORS doc/mdate.html doc/mdate.pdf doc/mdate.txt
-	install -d $manDir/man1 doc/mdate.1
+	install -t $manDir/man1 doc/mdate.1
```
No background color indicates the line is OK.
A red background (and the "-") indicates the line is incorrect.
A green background (and the "+") indicates what we expect, without the "+" in the beginning of the line, of course..

By the way, "install -d directory1 directory2 directory3..." is used to create the directories, while "install -t destinationdirectory file1 file2 file3 file4..." tells "install" that all the files should be installed in destinationdirectory.
Well, I guess you meant to use -t but forgot to fix yout copy/paste.